tools/workspace-plugin/src/executors/build/executor.spec.ts

```diff
@@ -263,5 +263,16 @@ describe('Build Executor', () => {
       });
       "
     `);
+
+    const publicApiFilePath = join(workspaceRoot, 'libs/proj/src/index.ts');
```

Comment:
executor was alread executed and didn't fail - we have api.md

following simulates the "out of date public api surface" which will fail the executor - mitigating the issue we faced